docs/quickstart-eks.md

```diff
@@ -0,0 +1,264 @@
+# Quickstart: BKPR on Amazon Elastic Container Service for Kubernetes (Amazon EKS)
+
+## TOC
+
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Installation and setup](#installation-and-setup)
+- [Next steps](#next-steps)
+  + [Installing Kubeapps on BKPR](kubeapps-on-bkpr.md)
+- [Upgrading](#upgrading)
+- [Teardown and cleanup](#teardown-and-cleanup)
+- [Further reading](#further-reading)
+
+## Introduction
+
+This document walks you through setting up an Amazon Elastic Container Service for Kubernetes (Amazon EKS) cluster and installing the Bitnami Kubernetes Production Runtime (BKPR) on it.
+
+## Prerequisites
+
+* [Amazon AWS account](https://aws.amazon.com/)
+* [Amazon CLI](https://aws.amazon.com/cli/)
+* [`eksctl`](https://aws.amazon.com/blogs/opensource/eksctl-eks-cluster-one-command/)
+* [Kubernetes CLI](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+* [BKPR installer](install.md)
+* [kubecfg](https://github.com/ksonnet/kubecfg/releases)
+* [jq](https://stedolan.github.io/jq/)
+
+### DNS requirements
+
+In addition to the requirements listed above, a domain name is also required for setting up Ingress endpoints to services running in the cluster. The specified domain name can be a top-level domain (TLD) or a subdomain. In either case you have to manually [set up the NS records](#step-3-configure-domain-registration-records) for the specified TLD or subdomain so as to delegate DNS resolution queries to an Amazon Route 53 hosted zone created and managed by BKPR.  This is required in order to generate valid TLS certificates.
+
+## Installation and setup
+
+### Step 1: Set up the cluster
+
+In this section, you will deploy an Amazon Elastic Container Service for Kubernetes (Amazon EKS) cluster using the `eksctl` tool`.
+
+* Make sure you have a working Amazon CLI environment. Please refer to [Configuring the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) if in doubt.
+
+* Configure the following environment variables:
+
+  ```bash
+  export BKPR_DNS_ZONE=my-domain.com
+  export AWS_EKS_USER=my-user@example.com
+  export AWS_EKS_CLUSTER=my-eks-cluster
+  export AWS_EKS_K8S_VERSION=1.11
+  ```
+
+  - `BKPR_DNS_ZONE` specifies the DNS suffix for the externally-visible websites and services deployed in the cluster.
+  - `AWS_EKS_USER` specifies the email address used in requests to Let's Encrypt.
+  - `AWS_EKS_CLUSTER` specifies the name of the AKS cluster.
+  - `AWS_EKS_K8S_VERSION` specifies the version of Kubernetes to use for creating the cluster. The [BKPR Kubernetes version support matrix](../README.md#kubernetes-version-support-matrix-for-bkpr-10) lists the base Kubernetes versions supported by BKPR.
```

Comment:
s/(../README.md#kubernetes-version-support-matrix-for-bkpr-10)/(../README.md#kubernetes-version-support-matrix-for-bkpr)

Reply:
Done.